cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -472,7 +472,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			}
 
 			bucket := targetOptions.Bucket
-			if impl.AWSBucket != "" {
+			if bucket == "" {
```

Comment:
@Gundersanne Is this fine for the service?

Reply:
Just FYI, we discussed this on Slack. The current logic is IMO weird, but I can live with it. I changed it to make it consistent with more appealing logic that I used for GCP. If we agree that we want to stick to the current logic used with AWS, I'll change it also for GCP, although I don't like it.

Reply:
Should be fine, the bucket is only specified in the worker.